packages/ui-keyring/src/Keyring.ts

```diff
@@ -82,6 +82,18 @@ export class Keyring extends Base implements KeyringStruct {
     return pair.toJson(password);
   }
 
+  public async backupAccounts (pairs: KeyringPair[]): Promise<KeyringPair$Json[]> {
```

Comment:
Sorry... commented in my head last night and then didn't actually put pen to paper.

Here it would actually make sense to only pass the addresses through, i.e. `string[]`.